tidb-cloud/import-csv-files.md

```diff
@@ -81,10 +98,33 @@ To import the CSV files to TiDB Cloud, take the following steps:
 
-    - **Target Database**: fill in the **Username** and **Password** fields.
-    - **DB/Tables Filter**: if necessary, you can specify a [table filter](/table-filter.md#syntax). If you want to configure multiple filter rules, use `,` to separate the rules.
-    - **Object Name Pattern**: enter a pattern that matches the names of the CSV files to be imported. For example,`my-data.csv`.
-    - **Target Table Name**: enter the name of the target table. For example, `mydb.mytable`.
+    - **Target Cluster**: fill in the **Username** and **Password** fields.
+    - **DB/Tables Filter**: if you want to filter which tables to be imported, you can specify one or more table filters in this field, separated by `,`.
+
+        For example:
+
+        - `db01.*`: all tables in the `db01` database will be imported.
+        - `db01.table01*,db01.table02*`: all tables starting with `table01` and `table02` in the `db01` database will be imported.
+        - `!db02.*`: except the tables in the `db02` database, all other tables will be imported. `!` is used to exclude tables that do not need to be imported.
+        - `*.*` : all tables will be imported.
+
+        For more information, see [table filter snytax](/table-filter.md#syntax).
+
+    - **Custom Pattern**: after enabling this feature, you can specify a custom mapping rule between multiple CSV files and a single target table. Enable this feature only if your CSV files do not follow the naming rules in [Step 1](#step-1-prepare-the-csv-files) and you want to import source data into the same target table.
+
+        > **Note:**
+        >
+        > - After enabling this feature, one import task can only import one table at a time. If you want to merge and import data into different tables, you need to import several times, each time specifying a different target table.
+        > - It is recommended that you first import the majority of tables using **DB/Tables Filter**. After that, start several more import tasks, specifying the custom mapping rules to import data to target tables one at a time.
+
+        When **Custom Pattern** is enabled, you are required to fill in the following fields:
+
+        - **Object Name Pattern**: enter a pattern that matches the names of the CSV files to be imported. For example:
+
+            - `my-data?.csv`: all CSV files starting with `my-data` and one character (such as `my-data1.csv` and `my-data2.csv`) will be imported into the same target table.
+            - `my-data*.csv`: all CSV files starting with `my-data` will be imported into the same target table.
+
+        - **Target Table Name**: enter the name of the target table in TiDB Cloud, which must be in the `${db_name}.${table_name}` format. For example, `mydb.mytable`. Note that this field accepts one specific table name only so wildcards are not supported.
```

Comment:
Should "only so"  be "only if"?

Reply:
What about "Note that this field only accepts one specific table name, so wildcards are not supported."?